src/components/NavBar.tsx

```diff
@@ -189,7 +187,7 @@ export default function NavBar({
             }}
             id='plants-dashboard'
           />
-          {trackingV2 && hasPlantingSites === true && (
+          {hasPlantingSites === true ? (
```

Comment:
This one's weird, it won't typecheck correctly without this change. I'm not sure why it was okay with the previous implementation

Reply:
I think maybe the type of the `SubNavBar` `children` is the culprit

Reply:
Got it, thanks for trying out the suggestion. We can follow up later maybe.  Looks good for now.